frontend/packages/console-shared/src/utils/multiselectdropdown.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+
+export const MultiSelectDropdown: React.FC<MultiSelectDropdownProps> = ({
+  onChange,
+  placeholder,
+  id,
+  options,
+  defaultSelected,
+}) => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState<string[]>(defaultSelected || []);
+  const onSelect = (event: React.MouseEvent | React.ChangeEvent, selection: string) => {
+    let cSelected: string[] = selected;
```

Comment:
This closure would cause this function to revaluated every time the value of selected is changed. And since `selected` is an array, I think we should create a memoized function using `useCallback` with `useDeepCompareMemoize`